src/Uno.UWP/Storage/StorageFolder.cs

```diff
@@ -174,6 +174,8 @@ internal StorageFolder(string name, string path)
                     return (IStorageItem)result;
                 });
 
+		public IAsyncOperation<StorageFile> CreateFileAsync(string path) => CreateFileAsync(path, CreationCollisionOption.FailIfExists);
```

Comment:
don't change the parameter name

Reply:
So, existing
public IAsyncOperation<StorageFile> CreateFileAsync(string path, CreationCollisionOption option)
should be corrected, to 
public IAsyncOperation<StorageFile> CreateFileAsync(String desiredName, CreationCollisionOption options)
?